projects/observability/src/shared/graphql/request/handlers/explore/explore-query.ts

```diff
@@ -35,7 +35,7 @@ export interface GraphQlExploreResult {
   [key: string]: GraphQlExploreResultValue;
 }
 
-type GraphQlExploreValueType = number | string | boolean | undefined;
+type GraphQlExploreValueType = number | string | string[] | boolean | undefined;
```

Comment:
is this intentional?

Reply:
yes. dataTypeIds and labels are all string array fields. The values are returned as string[]